project/schema_admin.py

```diff
@@ -0,0 +1,148 @@
+from typing import List, Optional
+import logging
+from functools import wraps
+from graphql import ResolveInfo
+from graphene_django.types import DjangoObjectType
+import graphene
+
+from project import schema_registry
+from users.models import CHANGE_USER_PERMISSION, JustfixUser
```

Comment:
I feel weird using the `CHANGE_USER_PERMISSION` here instead of the _view_ user permission, because none of the GraphQL endpoints that depend on it actually change anything.  It feels misleading.